elixir/cloudbuild-release.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 7200s  # 2 hours
+steps:
+# Elixir 1.14 build
+- name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/release-images/elixir114', '--build-arg', 'BASE_IMAGE=elixir:1.14', '--build-arg', 'SHORT_VERSION=114', 'elixir/']
+  id: elixir114-build
+  waitFor: [load-plts]
+- name: gcr.io/gcp-runtimes/container-structure-test
+  args: ['test', '--image', 'us-central1-docker.pkg.dev/$PROJECT_ID/release-images/elixir114', '--config', '/workspace/elixir/elixir114.yaml']
+  id: elixir114-test
+  waitFor: ['elixir114-build']
+
+images:
+- us-central1-docker.pkg.dev/$PROJECT_ID/release-images/elixir114
```

Comment:
This needs to include the load-plts step from elixir/cloudbuild.yaml

Reply:
Thanks for the feedback @dazuma i'll move this to a separate PR to not hold up the rest of the languages since this will required additional setup.